R/deploy-site.R

```diff
@@ -98,28 +96,24 @@ deploy_site_github <- function(
   cat_line("Setting private key permissions to 0600")
   fs::file_chmod(ssh_id_file, "0600")
 
-  deploy_local(pkg, repo_slug = repo_slug, host = host, commit_message = commit_message, ...)
+  deploy_local(pkg, commit_message = commit_message, ...)
 
   rule("Deploy completed", line = 2)
 }
 
-deploy_local <- function(
-                         pkg = ".",
-                         repo_slug = NULL,
-                         host,
+#' Build and deploy a site locally
```

Comment:
Probably worth updating the `deploy_site_github()` to mention `deploy_local()`. 

And maybe since we're exporting it, it's worth considering the name a bit more?

Reply:
I added a note about `depoly_local` to the `deploy_site_github` docs. I am happy to change the name, but I don't really have any suggestions, did you have something in mind?

Reply:
`deploy_to_branch()` maybe? `deploy_site_github()` isn't named very well either, unfortunately.

Reply:
Sure we could do `deploy_to_branch()`, we would just need to add an argument to set the branch (with a default to `gh-pages` I guess).